### PR TITLE
Season form post functionality

### PR DESCRIPTION
**Pull request by nathanSamosa**

### CODE REVIEW

**Review by polarbear23 — APPROVED** [review]

**Review by bravint — APPROVED** [review]

**Review by polarbear23 — APPROVED** [review]
Great work!

File: src/client/components/CreateSeason.jsx [hunk 1]
```diff
@@ -25,23 +36,68 @@ export const CreateSeason = () => {
...
+    }, [])
+
+    const postForm = async() => {
         console.log(form)
-    }
+        const fetchConfig = {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: token,
```
Comment (glowkeeper) [inline src/client/components/CreateSeason.jsx:66]: I'm not going to let this stop me approving the PR, but I _think_ it's good practice to include the string "Bearer" here, see: https://security.stackexchange.com/questions/108662/why-is-bearer-required-before-the-token-in-authorization-header-in-a-http-re - i.e:
```
Authorization: `Bearer ${token}`,
```

File: src/client/components/seasonform/TitleForm.jsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ export const TitleForm = ({ form, setForm, formStep, setFormStep }) => {
     return (
         <div className="form-title">
             <label>
-                Season Title
+                <h2>Season Title</h2>
```
Comment (glowkeeper) [inline src/client/components/seasonform/TitleForm.jsx:18]: Better might be to put all strings in a config file...

**Review by glowkeeper — APPROVED** [review]
Approved (although I _think_ I want to see _Bearer_ in those authorisation headers)

**Review by molloyconor511 — APPROVED** [review]

File: src/client/components/CreateSeason.jsx (hunk 1 above)
Comment (nathanSamosa) [reply src/client/components/CreateSeason.jsx:66]: Bearer is already included in the token when it is created
image

**Pull request merged**